.storybook/stories/useLoader.stories.js

```diff
@@ -0,0 +1,43 @@
+import { Vector3 } from 'three'
+import React, { Suspense } from 'react'
+import { Canvas, useFrame } from 'react-three-fiber'
+import ThorAndMidgardSerpent from '../public/scene_draco'
+import { useGLTFLoader } from '../../src/loaders/useGLTFLoader'
+import { Loader } from '../../src/prototyping/Loader'
+
+export default {
+  title: 'Prototyping/useLoader',
+  component: UseLoader,
+}
+
+function Suzanne() {
+  const { nodes } = useGLTFLoader('suzanne.glb', true)
+  return <mesh geometry={nodes.Suzanne.geometry} />
+}
+
+function ZoomIn() {
+  const vec = new Vector3(0, 10, 30)
+  return useFrame(({ camera }) => camera.position.lerp(vec, 0.1))
+}
+
+function UseLoader() {
+  return (
+    <>
+      <Loader />
```

Comment:
This should be put as a fallback for the <Suspense> block, check this example:

https://codesandbox.io/s/r3f-whats-new-in-drei-9ziry

The way you proposed, the loader would always be visible, even after we loaded everything

Reply:
Thanks for the feedback gsimone. I made the changes as you suggested, but I do not see the loader when I refresh the scene now. Can you try it and let me know? Thanks!